### ISSUE TITLE
evalsha() got an unexpected keyword argument 'keys'

### ISSUE THREAD

**Issue by PaiHsuehChung** [long2ice/fastapi-limiter#18]
HI:
   Fist of all, I want to appreciate that you build such awesome tool.
   When i use this library encounter some problem like title, which code write on 
```
 File "/usr/local/lib/python3.8/site-packages/fastapi_limiter/depends.py", line 42, in __call__
    pexpire = await redis.evalsha(
TypeError: evalsha() got an unexpected keyword argument 'keys'
```
  --- 
Environments:
- python 3.8
- aioredis == 2.0.0
- fastapi-limiter==0.1.4
- fastapi==0.68.1

**Comment by long2ice** [comment]
Upgrade fastapi-limiter

**Comment by ssinghgc** [comment]
Is this issue resolved? Still facing this.

**Comment by GGGoingdown** [comment]
@ssinghgc 
When i using pip or poetry to install still facing this problem. 
So you can build from source. It work for me (git+https://github.com/long2ice/fastapi-limiter.git)

**Comment by Me-Phew** [comment]
I am also facing the same issue (v. 0.1.4)

**Comment by MarcWarrior** [comment]
> Is this issue resolved? Still facing this.

You should upgrade fastapi-limiter to v0.1.4 or modify the original code like this:
```python
rate_key = await identifier(request)
key = f"{FastAPILimiter.prefix}:{rate_key}:{index}"
pexpire = await redis.evalsha(
    FastAPILimiter.lua_sha, 1, key, str(self.times), str(self.milliseconds)
)
````

**Comment by UnknownFX02** [comment]
@MarcWarrior fastapi-limiter to v0.1.4 still have this issue, but will fix with your snippets. Can you update this package to pipy

**Comment by charlax** [comment]
`fastapi-limiter-0.1.4.tar.gz` which is currently on pypi still has `keys` keyword argument. Can you push a new version?

Btw, https://github.com/aio-libs/aioredis-py is being merged into `redis-py`.

**Comment by Acendro** [comment]
I just came around the same issue and discovered a workaround that some of you might find helpful.
You can basically install the 1.0.6 version with: 'pip install https://github.com/hardbyte/fastapi-limiter/archive/patch-1.zip'
And in your requirements.txt just swap out the 'fastapi-limiter==xxx' to 'git+https://github.com/hardbyte/fastapi-limiter@patch-1'

Runs like a charm for me now =)

**Comment by WardPearce** [comment]
Yup is relevant to https://github.com/long2ice/fastapi-limiter/issues/20

Is the repo owner no longer maintaining this project?

**Comment by Me-Phew** [comment]
> Yup is relevant to https://github.com/long2ice/fastapi-limiter/issues/20
> 
> Is the repo owner no longer maintaining this project?

I don't think so

**Comment by Me-Phew** [comment]
> Yup is relevant to https://github.com/long2ice/fastapi-limiter/issues/20
> 
> Is the repo owner no longer maintaining this project?

I don't think he is

**Comment by python-programmer** [comment]
check your aioredis version, this version only works with aioredis < 2.0

for version 2.0 and above, I think the maintainer must change the below code:

```
  pexpire = await redis.evalsha(
      FastAPILimiter.lua_sha, keys=[key], args=[self.times, self.milliseconds]
  )
```

TO

```
keys = [key]
pexpire = await redis.evalsha(
  FastAPILimiter.lua_sha, len(keys), *keys, self.times, self.milliseconds
)
```
